opm/material/common/Tabulated1DFunction.hpp

```diff
@@ -272,7 +272,12 @@ class Tabulated1DFunction
         Scalar y0 = yValues_[segIdx];
         Scalar y1 = yValues_[segIdx + 1];
 
-        return y0 + (y1 - y0)*(x - x0)/(x1 - x0);
+        // Corner case where table is poorly defined
+        // with multiple y values for a single x value
+        if (x0 == x1)
```

Comment:
right: I don't care, so I'll just follow the judgement of the patch's author. I'm less lenient about identation, though.(which is fine here, BTW)